app/Daemon.hs

```diff
@@ -252,7 +254,8 @@ displayNotifications :: Maybe EwwWindow -> [Notification] -> IO ()
 displayNotifications w l =
   if not $ null l
     then do
-      let widgetString = buildWidgetWrapper True $ buildWidgetString l
+      let replaceNewlines = map (\c -> if c=='\n' then ' '; else c)
```

Comment:
The semicolon is redundant. Also I would prefer to move this function definition to Helpers.hs.